### ISSUE TITLE
Upsert operation doesn't return the expected status and/or body

### ISSUE THREAD

**Issue by kiskit** [ScorpioBroker/ScorpioBroker#329]
## Problem statement
A successful upsert operation on both existing and non existing entities does not return the correct body and/or status.
## Modus Operandi
First create an entity such as this one:
```json
[
	{
		"id": "uri:entity1",
		"type": "Temperature",
		"dataProvider": {
			"type": "Property",
			"value": "62349895462c09197b1d6776"
		},
		"xy": {
			"type": "Property",
			"value": "apopo"
		}
	}
]
```
Then use the upsert operation (/entityOperations/upsert) to modify the entity previously created and insert another one:
```json
[
	{
		"dataProvider": {
			"type": "Property",
			"value": "62349895462c09197b1d6776"
		},
		"id": "uri:entity1",
		"type": "Temperature"
	},
	{
		"dataProvider": {
			"type": "Property",
			"value": "62349895462c09197b1d6776"
		},
		"id": "uri:entity2",
		"type": "Temperature"
	}
]
```
## Expectations
In version "latest" or 2.1.22 (as pulled from the docker repository), the status returned is **207** and the body is **empty**. 
In version 2.1.13, the status returned is **207** and the body is a **BatchOperationResult**.
However, according to the specification (chapter 6.15), that case should warrant a **201** status, with a **list of all the entities created** returned in the body.

**Comment by amitrnec** [comment]
Hi,
Now fixed this issue #332 

Thanks.

**Comment by kiskit** [comment]
Great thanks
